kobotoolbox/kpi
# feat(organizations): manage member permissions on addition or removal via Django Admin TASK-965

### 📣 Summary
Manage member permissions when adding or removing members via the Django Admin interface, with automated data transfer on addition.


### 📖 Description
This feature ensures that permissions are correctly managed when members are added to or removed from an organization using the Django Admin interface. For added members, it automates the transfer of their data and project ownership to the organization, streamlining the onboarding process. For removed members, it ensures permissions are revoked to maintain data integrity and security. 


### Testing:
1. Create two accounts `user1` and `user2`
2. Log in as `user2`, add a project and data
3. In another browser window, log in as a superuser, Go to the admin interface and go to the user1's organization (Home > Organizations > Organizations , Click on first account organization)
4. Add `user2` in "Organization users" section
5. Save
6. Wait a moment (or look at the logs of celery worker which consumes the low prority queue) and reload the project list (logged in as `user2`). Owner of the project should have changed to organization's owner
7. From the admin interface, remove `user2` from `user1's organization` 
8. Refresh the project list (logged in as `user2`). The project which belongs to user1's organization should not be there anymore
